docs/guides/continuous-integration/github-actions.mdx

```diff
@@ -111,8 +111,8 @@ jobs:
 
 To try out the example above yourself, fork the
 [Cypress Kitchen Sink](https://github.com/cypress-io/cypress-example-kitchensink)
-example project and place the above GitHub Action configuration in
-`.github/workflows/main.yml`.
+example project and place the above GitHub Action configuration in a newly
```

Comment:
I think the previous verbiage implied this, but I could see a newer engineer might not catch the nuance.

Reply:
Yeah, that was the goal here. GH Actions seems to be the go-to for many engineers since the ramp-up to get started is already part of their workflow.